### PR TITLE
Core: English documentation for all functions

### PR DESCRIPTION
**Pull request by HollowMan6**
Please review and give me some suggestions. Mistakes may happen from time to time.

Thanks a lot!

### CODE REVIEW

**Comment by manateelazycat** [comment]
thanks for patch man

File: eaf.el [hunk 1]
```diff
@@ -1073,6 +1075,7 @@ to edit EAF keybindings!" fun fun)))
    "is_support"
    url))
 
+;; Update eaf view once emacs window size changed.
```
Comment (MatthewZMD) [inline eaf.el:1078]: 这个放到函数里面吧

File: eaf.el [hunk 2]
```diff
@@ -1311,6 +1320,7 @@ of `eaf--buffer-app-name' inside the EAF buffer."
  #'eaf--create-new-browser-buffer)
 
 (defun eaf--create-new-browser-buffer (new-window-buffer-id)
+  "Function for creating a new browser buffer with the specified window buffer id."
```
Comment (MatthewZMD) [inline eaf.el:1323]: 按照Emacs的elisp函数要求，第一行需要函数parameter的全大写

File: eaf.el (hunk 1 above)
Comment (HollowMan6) [reply eaf.el:1078]: 好的！

File: eaf.el (hunk 2 above)
Comment (HollowMan6) [reply eaf.el:1323]: OK，了解，我稍后会按照此要求进行更改，重新发一个PR

**Pull request merged**